### PR TITLE
Migrated docs to Wiki and deleted doc file as it is redundant to the wiki

### PR DESCRIPTION
**Pull request by austinprog**
#68 Just one part of what I also hope to contribute to the documentation now that I finally have time to help out on this project once more.

### CODE REVIEW

**Comment by drewbrew** [comment]
Once this PR is good, I'll clone and push your wiki docs to this repo so you don't have to copy/paste everything.

One minor note: in #202, I made some minor updates to the readme that look like they didn't make it into your wiki. No big deal, just something to be aware of so we don't lose info in the transition.

**Comment by austinprog** [comment]
Thanks! I shall have those changes done soon.

**Comment by austinprog** [comment]
Alright. I did the changes requested correctly hopefully.

File: docker-compose.yml [hunk 1]
```diff
@@ -20,9 +20,6 @@ services:
       - "8000:8000"
     depends_on:
       - postgres
-  documentation:
-    restart: always
-    build: ./
     command: >
```
Comment (drewbrew) [inline docker-compose.yml]: You need to delete [lines 23-28](https://github.com/hsv-dot-beer/hsvdotbeer/blob/cbefb61df7763d547d2f95150fee7a15cfe95c58/docker-compose.yml#L23-L28) too

**Comment by drewbrew** [comment]
Looks perfect. I'm going to wait to merge until I get back to my laptop tomorrow to take one more look at your wiki.

**Pull request merged**